AbstractRendererTestBase not usable if PrimeFaces renderers are extended and should be tested

org.apache.myfaces.test.mock.MockServletContext throw UnsupportedOperationException if getSessionCookieConfig get called.
```
Caused by: java.lang.UnsupportedOperationException: Not supported yet.
	at org.apache.myfaces.test.mock.MockServletContext.getSessionCookieConfig(MockServletContext.java:697)
	at org.primefaces.config.PrimeConfiguration.<init>(PrimeConfiguration.java:160)
	at org.primefaces.context.PrimeApplicationContext.<init>(PrimeApplicationContext.java:80)
	at org.primefaces.context.PrimeApplicationContext.getCurrentInstance(PrimeApplicationContext.java:232)
	at org.primefaces.context.PrimeApplicationContext.getCurrentInstance(PrimeApplicationContext.java:220)
	at org.primefaces.util.AjaxRequestBuilder.resetValues(AjaxRequestBuilder.java:243)
	at org.primefaces.renderkit.CoreRenderer.preConfiguredAjaxRequestBuilder(CoreRenderer.java:466)
	at org.primefaces.renderkit.CoreRenderer.buildAjaxRequest(CoreRenderer.java:481)
	at org.primefaces.renderkit.CoreRenderer.buildAjaxRequest(CoreRenderer.java:477)
```

Therefore CuiMockServletContext could provide SessionCookieConfig

--- a/src/main/java/de/cuioss/test/jsf/mocks/CuiMockServletContext.java
+++ b/src/main/java/de/cuioss/test/jsf/mocks/CuiMockServletContext.java
@@ -18,12 +18,14 @@ package de.cuioss.test.jsf.mocks;
 import java.nio.charset.StandardCharsets;
 
 import javax.servlet.ServletRegistration.Dynamic;
+import javax.servlet.SessionCookieConfig;
 
 import org.apache.myfaces.test.mock.MockServletContext;
 
 import lombok.Getter;
 import lombok.Setter;
 
+@Getter
 public class CuiMockServletContext extends MockServletContext {
 
     @Override
@@ -31,24 +33,22 @@ public class CuiMockServletContext extends MockServletContext {
         throw new UnsupportedOperationException();
     }
 
-    @Getter
     @Setter
     private int sessionTimeout = 200;
 
-    @Getter
     @Setter
     private String requestCharacterEncoding = StandardCharsets.UTF_8.name();
 
-    @Getter
     @Setter
     private String responseCharacterEncoding = StandardCharsets.UTF_8.name();
 
-    @Getter
     @Setter
     private String virtualServerName = "virtual";
 
-    @Getter
     @Setter
     private String contextPath = "mock-context";
+
+    @Setter
+    private SessionCookieConfig sessionCookieConfig;
 
 }

--- a/src/test/java/de/cuioss/test/jsf/mocks/CuiMockServletContextTest.java
+++ b/src/test/java/de/cuioss/test/jsf/mocks/CuiMockServletContextTest.java
@@ -15,12 +15,11 @@
  */
 package de.cuioss.test.jsf.mocks;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-
 import java.nio.charset.StandardCharsets;
 
 import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
 
 class CuiMockServletContextTest {
 
@@ -33,6 +32,7 @@ class CuiMockServletContextTest {
         assertEquals(200, context.getSessionTimeout());
         assertEquals("virtual", context.getVirtualServerName());
         assertEquals("mock-context", context.getContextPath());
+        assertNull(context.getSessionCookieConfig());
         assertThrows(UnsupportedOperationException.class, () -> context.addJspFile("aa", "bb"));
     }
 

